src/components/learner-credit-management/members-tab/MembersTableSwitchFilter.jsx

```diff
@@ -0,0 +1,32 @@
+import { Form } from '@edx/paragon';
+import PropTypes from 'prop-types';
+
+const MembersTableSwitchFilter = ({ column: { filterValue, setFilter } }) => (
+  <Form.Switch
+    className="ml-2.5"
```

Comment:
nit: it might look a bit better if it's vertically centered, so adding mt-2.5 to this